.github/workflows/build-test-measure.yml

```diff
@@ -491,12 +534,6 @@ jobs:
         if: needs.pre-run.outputs.changed-php-count > 0
         run: bash bin/ci/after-wp-install.sh ${{ matrix.wp }} ${{ matrix.install-pwa-plugin == true }}
 
-      - name: Setup PCOV
-        if: ${{ matrix.coverage == true && needs.pre-run.outputs.changed-php-count > 0 }}
-        run: |
-          composer require --dev --ignore-platform-reqs pcov/clobber
-          vendor/bin/pcov clobber
-
```

Comment:
I think pcov is disabled temporarily, right?

Reply:
PCOV support was merged into PHPUnit 8. `pcov/clobber` is no longer needed.